src/components/organisms/achievements/achievements.tsx

```diff
@@ -0,0 +1,31 @@
+import { type FC } from 'react';
+import { Achievement } from 'components/molecules/achievement';
+import { useAppSelector } from 'store';
+import { selectAchievements } from 'store/achievements/selectors';
+import { AchievementsProps } from './types';
+import styles from './achievements.module.scss';
+
+/**
+ * Компонент блока ачивок и всплывающего окна - tooltip
+ */
+
+export const Achievements: FC<AchievementsProps> = () => {
+  const dataAchievements = useAppSelector(selectAchievements);
```

Comment:
```suggestion
  const achievements = useAppSelector(selectAchievements);
```

Если при запросе случится ошибка или ачивок не будет, то на странице отрендерится пустой див (сейчас даже три). Нужно добавить проверку на длину массива и возвращать из компонента `null`, если массив пустой.